client/my-sites/email/email-management/home/email-mailbox-action-menu.jsx

```diff
@@ -333,75 +333,14 @@ RemoveTitanMailboxConfirmationDialog.propTypes = {
 	setVisible: PropTypes.func.isRequired,
 };
 
-const EmailForwardRemovalResponseMessage = ( { mailbox, successful } ) => {
```

Comment:
I moved this logic from the action menu to the react-query mutation because the optimistic removal of email forwards would otherwise cause this notification to not appear at all.